package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "action-setup-enos",
-  "version": "1.30",
+  "version": "1.31",
```

Comment:
I'm not totally familiar with these files, but is it intended for this version to be misaligned with the version in `package.json`?

Reply:
That is the version of the _action_, the version of enos that we install is handled differently. Suppose we wanted to cut a v2 of the action, this would go to 2.0 and increment from there.